.github/stale.yml

```diff
@@ -0,0 +1,20 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 90
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
```

Comment:
do we need something here or is it a valid key empty like this? maybe we can just remove?

Reply:
Adjusted